doc/conf.py

```diff
@@ -21,6 +22,82 @@
 if not os.path.islink('.sphinx/_static/swagger-ui/swagger-ui.css'):
     os.symlink('../../deps/swagger-ui/dist/swagger-ui.css', '.sphinx/_static/swagger-ui/swagger-ui.css')
 
+### MAN PAGES ###
+
+# Download lxc client
+
+if not os.path.isfile('.sphinx/deps/bin.linux.lxc'):
```

Comment:
Is there somewhere else we can move the downloading of dependencies out to, like a github workflow or a readthedocs hook, as I'd like to avoid building the docs from a version of the client that is different to the one running locally.

Reply:
As discussed - let's change to using `lxc` for local builds (the tool should be in the path if the environment is set up correctly) and building the tool on RTD (see https://github.com/canonical/lxd/blob/main/.github/workflows/tests.yml#L274 ).